src/button/Button.module.css

```diff
@@ -207,3 +207,10 @@ limitations under the License.
 .lg {
   height: 40px;
 }
+
+.linkButton {
+  background-color: transparent;
+  border: none;
+  text-decoration: underline;
+  cursor: pointer;
+}
```

Comment:
I think what our design systems have to say about this is that link buttons should be green with no underline, like in Element Web